index.html

```diff
@@ -263,7 +270,67 @@ <h3 class="about-title">Skills</h3>
         </div>
       </div>
     </div>
-      </section>
-    </main>
+    <div class="contact-form">
+      <nav class="text-layout">
```

Comment:
- [ ] Kindly, use `footer` tag instead of `div` tag to follow the [HTML semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp) due to SEO purpose :blush:

Reply:
Done